src/doc/book/unsized-types.md

```diff
@@ -55,6 +55,7 @@ struct Foo<T: ?Sized> {
 }
 ```
 
-This `?Sized`, read as “T may or may not be `Sized`”, allowing us to match both
-constant size and unsized types. All generic type parameters implicitly have
-the `Sized` bound, so `?Sized` can be used to opt-out of the implicit bound.
+This `?Sized`, read as “T may or may not be `Sized`”, which allows us to match
```

Comment:
Sorry, I think this is still not quite right...
My understanding is that you should be able to remove the appositive (the part between the commas) and still have a valid sentence. So I think the word "which" here is extraneous.

I'd prefer if someone else could chime in here as well on the grammar.